docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,1714 @@
+{
```

Comment:
<p>FYI, I am working on a way to parallelize <code>prescrump</code> in <a href="https://github.com/TDAmeritrade/stumpy/issues/551" target="_blank">this issue</a>. I'll let you know when that is complete and then you may need to update this function in the future.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
Cool! I also noticed you mention new `core._sliding_dot_product`. So, I assume I should change that as well. correct?

Reply:
<p>Yes but let's wait because there might be a lot of changes that need to get figured out first.</p>

Reply:
@seanlaw 
I know you said "wait", but I had to change `_prescrump()` as I got error when I ran the code with its current design. I noticed that you changed the arguments of this function. Should I still wait? (or since you already close the issue, I can use the the new `core._sliding_dot_product` as well?

Reply:
@NimaSarajpoor I think things should be stable now. You should be able to pull the latest version into your code and please update the calls to `_prescrump` and `core._sliding_dot_product`.

Reply:
I will apply the changes.